src/server/common/model/state/state-manager.test.js

```diff
@@ -54,43 +54,29 @@ describe('StateManager.toState', () => {
     const partialState = { origin }
     const request = testRequest(partialState)
     const state = new StateManager(request)
-    expect(state.toState()).toEqual(partialState)
+    expect(state.toState()).toStrictEqual(partialState)
```

Comment:
I think we want to also test that request.yar.get was called with `application`?

Reply:
done